.github/workflows/dispatch_documentation.yml

```diff
@@ -0,0 +1,19 @@
+name: Trigger Doc Build
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  workflow_trigger:
+    runs-on: ubuntu-latest
+
+ steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Dispatch Event to Trigger Docs Build
+        uses: GitHub/dispatch@v2
```

Comment:
I cannot find this action online, is it a placeholder? This seems like the equivalent event https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#repository_dispatch - there's also [workflow_dispatch](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch), but I think the first one would be the one we'd need to use here

Reply:
Yes, it's a placeholder, should have clarified!